dvc/external_repo.py

```diff
@@ -3,30 +3,24 @@
 import tempfile
 import threading
 from contextlib import contextmanager
-from typing import Dict, Iterable
+from typing import Dict
 
-from funcy import cached_property, reraise, retry, wrap_with
+from funcy import cached_property, retry, wrap_with
 
 from dvc.exceptions import (
-    DvcException,
     FileMissingError,
     NoOutputInExternalRepoError,
     NoRemoteInExternalRepoError,
     NotDvcRepoError,
     OutputNotFoundError,
     PathMissingError,
 )
-from dvc.path_info import PathInfo
 from dvc.repo import Repo
 from dvc.utils import relpath
 
 logger = logging.getLogger(__name__)
 
 
-class IsADVCRepoError(DvcException):
-    """Raised when it is not expected to be a dvc repo."""
-
```

Comment:
This was really an unnecessary limitation. If we can `dvc list` the repo, we should be able to get/import it too.